Habera - update from docx

finsberg opened the pull request:
Add version from the word document

@michalhabera Could you please check that everything looks good and give a 👍 if you agree with the changes.

michalhabera commented on chapters/habera/main.tex:
```diff
@@ -226,12 +227,12 @@ \subsubsection*{Generated Code Structure}
 
 The most performance-critical part of the code is contained in the quadrature
-loop body. For the eighth-order Laplace operator, \lstinline{NUM_QUAD_POINTS = 214} and \lstinline{NUM_DOFS = 165}. There are two
+loop body. For the eight order Laplace operator, \lstinline{NUM_QUAD_POINTS = 214} and \lstinline{NUM_DOFS = 165}. There are two
```
```suggestion
loop body. For the eighth-order Laplace operator, \lstinline{NUM_QUAD_POINTS = 214} and \lstinline{NUM_DOFS = 165}. There are two
```

michalhabera commented on chapters/habera/main.tex:
```diff
@@ -170,19 +171,19 @@ \subsection*{Finite Element Kernels}
 The generated kernel calculates a double-precision vector $v_i \in
 \mathbb{R}^{n}$, where $n = 4$ for first-order (low-order) discretization and
-$n = 165$ for eighth-order (high-order) discretization. Low-order kernels are
+$n = 165$ for eighth-order discretization (high-order) discretization. Low-order kernels are
```
```suggestion
$n = 165$ for eighth-order (high-order) discretization. Low-order kernels are
```

finsberg commented:
> Thanks, this is the diff between all changes from the Word accepted and the Word document I've sent per email, right? Spotted few minor issues, added fixes as suggestions.

Yes exactly. Thanks for the suggestions - they are now incorporated.

michalhabera reviewed (approved):
All good. I got lost in the Word review process at places. This is so much more clear in github...
